sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -433,6 +456,73 @@ Construct a SYCL buffer instance from a pointer to a Level Zero memory allocatio
 description above for semantics and restrictions.
 The additional <code>AvailableEvent</code> argument must be a valid SYCL event. The instance of the SYCL buffer class template being constructed must wait for the SYCL event parameter to signal that the memory native handle is ready to be used.
 </tr>
+
+<tr>
+<td>
+
+``` C++
+template<backend Backend, int Dimensions = 1, typename AllocatorT = sycl::image_allocator>
+image<Dimensions, AllocatorT> make_image(
+    const backend_input_t<Backend, image<Dimensions, AllocatorT>> &backendObject,
+    const context &targetContext);
+```
+</td>
+<td>This API is available starting with revision 5 of this specification.
+
+Construct a SYCL image instance from a ze_image_handle_t. 
+
+Because LevelZero has no way of getting image information from an image, it must be provided.  The  <code>backend_input_t</code> is a struct type like so:
+``` C++
+struct type {
+    ze_image_handle_t ZeImageHandle;
+    sycl::image_channel_order ChanOrder;
+    sycl::image_channel_type ChanType;
+    sycl::range<Dimensions> Range;
+    ext::oneapi::level_zero::ownership Ownership{
+        ext::oneapi::level_zero::ownership::transfer};
+  };
+```
+where the Range should be ordered (width), (width, height), or (width, height, depth) for 1D, 2D and 3D images respectively, 
+with those values matching the dimensions used in the `ze_image_desc` that was used to create the `ze_image_handle_t` initially. 
+Note that the range term ordering (width first, depth last) is true for SYCL 1.2.1 images that are supported here. But future classes like
+sampled_image and unsampled_image might have a different ordering.
+
+Example Usage
+``` C++
+sycl::backend_input_t<BE, sycl::image<2>> ImageInteropInput{ 
+    ZeHImage, 
+    ChanOrder,
+    ChanType, 
+    ImgRange_2D, 
+    sycl::ext::oneapi::level_zero::ownership::transfer };      
+    
+sycl::image<2> Image_2D  
+  = sycl::make_image<BE, 2>(ImageInteropInput, Context);
+```
+
+ The input SYCL context <code>Context</code> must be associated with a single device, matching the device used to create the Level Zero image handle.

```

Comment:
> must be associated with a single device, matching the device used to create the Level Zero image handle.

NIT: this is an overkill to require that the context has that single device only. It is enough that the image is only used on the device where it was created, but not on other devices on the same context (limitation that exists today, but can be relaxed in future)